app/api/v2/views/meetups_view.py

```diff
@@ -1,13 +1,16 @@
 from flask import Blueprint, Flask, jsonify, request
 
 from app.api.v2.models.meetups_model import MeetUpsModel
+from app.api.v2.models.meetups_model import UsersModel
 
 meetupV2 = Blueprint('meetup_v2', __name__, url_prefix='/api/v2')
 
 meetups_model = MeetUpsModel()
+users_model = UsersModel()
 
 
 @meetupV2.route('/meetups', methods=['POST'])
+#@users_model.token_required
```

Comment:
Do not push commented code. If you are not using it then remove it.

Reply:
I've Implemented the changes recommended on branch name bg-fix-failing-build-163348045